phpstan.neon.dist

```diff
@@ -1,4 +1,13 @@
+includes:
+    - vendor/szepeviktor/phpstan-wordpress/extension.neon
 parameters:
-	level: 5
-	paths:
-		- src
+#    level: 5
+    level: 0
+    scanFiles:
+        # fullUrl() comes from this file
+        - vendor/getpop/migrate-engine/migrate/library/functions/utils.php
+    paths:
+        - src/
+    ignoreErrors:
+        # TODO Tidy up abstract methods.
+        - '#^Call to an undefined method GraphQLAPI\\GraphQLAPI\\Blocks\\AbstractBlock::getScriptDependencies\(\)\.$#'
```

Comment:
I have fixed this error in `master`, removing `getScriptDependencies`. Please sync your branch and re-execute phpstan

Reply:
Done.